cmd/mcu_com.c

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: GPL-2.0+
+
+#include <common.h>
+#include <command.h>
+#include <console.h>
+
+__weak int board_mcu_set_cpu_ready(void)
+{
+	return 0;
+}
+
+static int do_mcu_communications(cmd_tbl_t * cmdtp, int flag, int argc, char * const argv[])
+{
+	const char *command;
+	int ret;
+
+	if (argc < 2)
+		return CMD_RET_USAGE;
+
+	command = argv[1];
+
+	switch (*command) {
+	case 'c':
+		ret = board_mcu_set_cpu_ready();
+		if (ret < 0)
+			printf("Set CPU_READY failed");
+		break;
+	default:
+		return CMD_RET_USAGE;
+	}
+
+	if (ret < 0)
+		return CMD_RET_FAILURE;
+
+	return CMD_RET_SUCCESS;
+}
+
+#if CONFIG_IS_ENABLED(SYS_LONGHELP)
+static char mcu_comm_help_text[] =
+	"bootmap <cpu_ready>\n"
```

Comment:
`bootmap`??? 😕

Reply:
I also don't know, taken from 9f0e582239916661ca3faa64ca9a7788f80990d9

Reply:
As I guess your PR will be merged firs, I kindly ask you to take care of it